radio/src/targets/horus/CMakeLists.txt

```diff
@@ -80,6 +112,16 @@ if (PCB STREQUAL X10)
     option(INTERNAL_MODULE_MULTI "Support for MULTI internal module" OFF)
     option(BLUETOOTH "Support for bluetooth module" OFF)
     add_definitions(-DMANUFACTURER_JUMPER)
+# //OW
+    if (TELEMETRY_MAVLINK)
+      option(INTERNAL_GPS "Support for internal ublox GPS" OFF)
+      set(AUX_SERIAL ON)
+      if (NOT BLUETOOTH AND NOT INTERNAL_GPS)
+        set(AUX2_SERIAL ON)
```

Comment:
This definition belongs to each radio, it is not advisable to set it by yourself. Same applies to `AUX_SERIAL`.

Reply:
> This definition belongs to each radio, it is not advisable to set it by yourself. Same applies to `AUX_SERIAL`.

If i remember right. This was the same way 3djc activated the second serial on the tx16s a while ago